external-crates/move/crates/move-compiler/src/linters/unnecessary_unit.rs

```diff
@@ -66,8 +41,7 @@ impl TypingVisitorContext for Context<'_> {
             }
         }
         false
-    }
-
+    },
```

Comment:
rustfmt adds the , here regardless of the declaration syntax when using parens at the invocation